features/latest_version_features/posting_a_job.feature

```diff
@@ -3,11 +3,12 @@ Feature: Posting a job
   Scenario: Job is posted successfully
 
     Given the search api is working correctly
-    And set the api version to undefined for incoming requests
+    Given the api version is undefined for incoming requests
```

Comment:
How does the `Given` on line 6 interact with the `Given` on line 5?

Reply:
Good spot. Addressed